falcon_multiqc/commands/batch.py

```diff
@@ -11,8 +11,8 @@ def cli():
 
     with session_scope() as session:
         click.echo("Current list of batches in database are:\n")
-        batch_list = session.query(*[c for c in Batch.__table__.c if c.name != 'path']).all()
-        [click.echo(f"Batch_ID {b[0]}:      Details (cohortID {b[1]}, flowCell {b[2]}, date {b[3]})") for b in batch_list] 
+        batch_list = session.query(Batch.id, Batch.cohort_id, Batch.batch_name, Batch.description).all()
```

Comment:
just added this too, it's just cleaner and probably better

Reply:
nice im gonna use the objects __rep__ thing to print